### PR TITLE
fix for 2024b and update default to 2024a

### PR DESCRIPTION
**Pull request by ruslandoga**
temp fix for #143 

---

> update default to 2024a

Somehow https://data.iana.org/time-zones/tzdata-latest.tar.gz returns [2024a](https://github.com/lau/tzdata/issues/143#issuecomment-2333412098) for me. Also https://github.com/lau/tzdata/pull/144#issuecomment-2333451218

### CODE REVIEW

File: mix.lock [hunk 1]
```diff
@@ -10,6 +10,6 @@
   "mimerl": {:hex, :mimerl, "1.2.0", "67e2d3f571088d5cfd3e550c383094b47159f3eee8ffa08e64106cdf5e981be3", [:rebar3], [], "hexpm", "f278585650aa581986264638ebf698f8bb19df297f66ad91b18910dfc6e19323"},
   "nimble_parsec": {:hex, :nimble_parsec, "1.1.0", "3a6fca1550363552e54c216debb6a9e95bd8d32348938e13de5eda962c0d7f89", [:mix], [], "hexpm", "08eb32d66b706e913ff748f11694b17981c0b04a33ef470e33e11b3d3ac8f54b"},
   "parse_trans": {:hex, :parse_trans, "3.3.0", "09765507a3c7590a784615cfd421d101aec25098d50b89d7aa1d66646bc571c1", [:rebar3], [], "hexpm", "17ef63abde837ad30680ea7f857dd9e7ced9476cdd7b0394432af4bfc241b960"},
-  "ssl_verify_fun": {:hex, :ssl_verify_fun, "1.1.6", "cf344f5692c82d2cd7554f5ec8fd961548d4fd09e7d22f5b62482e5aeaebd4b0", [:make, :mix, :rebar3], [], "hexpm", "bdb0d2471f453c88ff3908e7686f86f9be327d065cc1ec16fa4540197ea04680"},
+  "ssl_verify_fun": {:hex, :ssl_verify_fun, "1.1.7", "354c321cf377240c7b8716899e182ce4890c5938111a1296add3ec74cf1715df", [:make, :mix, :rebar3], [], "hexpm", "fe4c190e8f37401d30167c8c405eda19469f34577987c76dde613e838bbc67f8"},
```
Comment (ruslandoga) [inline mix.lock:13]: https://github.com/deadtrickster/ssl_verify_fun.erl/pull/27

**Comment by ruslandoga** [comment]
Seems like there are more problems with 2024b. It fails to identify offsets in some datetimes:

```
on 2024b:  #DateTime<2015-01-13 07:00:07-12:00 %z Etc/GMT+12>
on 2024a: #DateTime<2015-01-13 07:00:07-12:00 -12 Etc/GMT+12>
```

**Comment by lau** [comment]
> Seems like there are more problems with 2024b. With it, tzdata fails to identify zone_abbr in some datetimes:

Thanks for mentioning that. This should be fixed in the recently released tzdata version 1.1.2.

**Comment by ruslandoga** [comment]
@lau thank you for the fixes!

**Pull request closed without merging**